subsys/net/ip/net_tc.c

```diff
@@ -259,23 +298,33 @@ void net_tc_tx_init(void)
 
 		NET_DBG("[%d] Starting TX queue %p stack size %zd "
 			"prio %d %s(%d)", i,
-			&tx_classes[i].work_q,
+			&tx_classes[i].fifo,
```

Comment:
Not sure on this one and it's a nit really, but shouldn't it rather report the `handler` thread address? All of the other debug info in this log corresponds to the actual thread, not the fifo structure itself.

Reply:
Yeah, probably the fifo address is not very important here. Will fix.

Reply:
> Yeah, probably the fifo address is not very important here. Will fix.

Hmm, actually printing the handler address is kind of pointless because all the threads use the same handler thread.

Reply:
I will leave the printing of the fifo here at this point, the debug print can be changed later as needed.

Reply:
> Hmm, actually printing the handler address is kind of pointless because all the threads use the same handler thread.

Are they? Yes, they will all use the same function as an entry point, but the actual thread address should be different (I've meant `struct k_thread handler;` from the `struct net_traffic_class`, probably the naming is a bit confusing).

Reply:
Ah, you meant that address, ok that makes sense indeed.